bears/r/FormatRBear.py

```diff
@@ -17,7 +15,40 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        rcode = ("formatR::tidy_source(source=" + "\""+filename+"\")")
+    def create_arguments(filename, file, config_file,
+                         comment: str="TRUE",
+                         blank: str="TRUE",
+                         brace: str="FALSE",
+                         arrow: str="FALSE",
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         max_line_length: int=80,
+                         output: str="TRUE",
+                         output_file: str=""):
```

Comment:
1. please rename many of these settings that are bound to R only, so something like `r_blank` (and maybe even a bit more information, I don't know for example what `blank` is meaning)
2. remove output file and output. the point of coala is to catch the output, we don't want to delegate that
3. it seems your `blank`, `brace` and other settings are actual booleans. Please use `bool`, and for the formatting below do `"TRUE" if setting else "FALSE"`